### PR TITLE
Add base structure and first contents

### PR DESCRIPTION
**Pull request by FriederikeM**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (hugh-burgess) [inline index.html:5]: someone's worked for CIA 😃

Comment (hugh-burgess) [reply index.html:5]: LGTM!

**Review by mpagels — APPROVED** [review]
Very good Job!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        exercitationem dolorum, atque laborum quos velit provident voluptatem
+        obcaecati, rem soluta repellendus distinctio eum voluptas placeat dicta
+        beatae. Maiores doloribus vitae quo iure quibusdam voluptates. Ducimus,
+        neque error qui reprehenderit modi facilis repellat tempore numquam eius
+        beatae ipsa inventore id vitae, saepe quod illo earum quae maiores!
+        Adipisci laborum cumque sunt quo impedit soluta ipsam.
+      </p>
+    </section>
+    <a href="#top"> Back to top </a>
+  </body>
```
Comment (namirsab) [inline index.html:97]: Looking good @FriederikeM !
Maybe you want to add a footer with some information? Just a suggestion ☺️

**Review by namirsab — APPROVED** [review]
Looking good! Congrats!

**Review by soerendolling — COMMENTED** [review]
lgtm

**Review by Alexander-Schulte — COMMENTED** [review]
Looks very good! Especially the <head> part, wow! 🤩

**Pull request merged**